app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt

```diff
@@ -135,14 +133,14 @@ private fun OngoingCallContent(
 ) {
 
     val sheetInitialValue =
-        if (classificationType == SecurityClassificationType.NONE) BottomSheetValue.Collapsed else BottomSheetValue.Expanded
-    val sheetState = rememberBottomSheetState(
+        if (classificationType == SecurityClassificationType.NONE) SheetValue.PartiallyExpanded else SheetValue.Expanded
+    val sheetState = rememberStandardBottomSheetState(
         initialValue = sheetInitialValue
     ).also {
-        LaunchedEffect(Unit) {
-            // Same issue with expanded on other sheets, we need to use animateTo to fully expand programmatically.
-            it.animateTo(sheetInitialValue)
-        }
+//        LaunchedEffect(Unit) {
+//            // Same issue with expanded on other sheets, we need to use animateTo to fully expand programmatically.
+//            it.show()
+//        } TODO
```

Comment:
Not needed ? maybe we remove it if so

Reply:
Oh, I forgot, that's why I'm adding these "TODO"s to code but this time when committing I didn't get the "TODO" warning, don't know why, so it slipped through 🤔 I commented that at the beginning as I wasn't sure how the "show" should be implemented in material3, but I fixed that already.